src/main/kotlin/uk/gov/dluhc/notificationsapi/mapper/NotificationTypeMapper.kt

```diff
@@ -49,6 +49,8 @@ interface NotificationTypeMapper {
     @ValueMapping(source = "REJECTED_SIGNATURE", target = "REJECTED_MINUS_SIGNATURE")
     @ValueMapping(source = "REQUESTED_SIGNATURE", target = "REQUESTED_MINUS_SIGNATURE")
     @ValueMapping(source = "NINO_NOT_MATCHED", target = "NINO_MINUS_NOT_MINUS_MATCHED")
+    @ValueMapping(source = "REJECTED_PARENT_GUARDIAN", target = "REJECTED_MINUS_PARENT_MINUS_GUARDIAN")
```

Comment:
We should update the test case in NotificationTypeMapperTest with these new cases